### PR TITLE
Add shelly documentation

### PR DESCRIPTION
**Pull request by ispiropoulos**
**Description:**
Documentation for the shelly switch component

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#16268

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Review by DubhAd — DISMISSED** [review]
👍

File: source/_components/switch.shelly.markdown [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+layout: page
+title: "Shelly Wifi Switch"
+description: "Instructions for the Shelly.cloud Wifi switch"
+date: 2018-08-28 11:42
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Switch
+ha_release: 0.76
```
Comment (frenck) [inline source/_components/switch.shelly.markdown]: 0.76 has been ship a while back already, please update the version number to the next major release.

File: source/_components/switch.shelly.markdown [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+```yaml
+# Example configuration.yaml entry
+switch:
+  platform: shelly
+  switches:
+    living_room_light:
+      host: hostname_or_ipaddr
+```
+
+Configuration variables:
```
Comment (frenck) [inline source/_components/switch.shelly.markdown:31]: Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
@ispiropoulos I've reviewed your PR and left you some comments. Could you please take a look? 👍

**Comment by zsarnett** [comment]
@ispiropoulos

File: source/_components/switch.shelly.markdown (hunk 2 above)
Comment (frenck) [reply source/_components/switch.shelly.markdown:31]: @ispiropoulos Not sure why you've marked this comment as "Resolved", while it is not.
Please convert the configuration variables section as per link listed above.

**Comment by frenck** [comment]
Closing this PR, since the parent PR has been closed.

**Pull request closed without merging**